superset-frontend/src/dashboard/components/nativeFilters/FilterBar/FilterSets/FilterSets.tsx

```diff
@@ -66,27 +77,59 @@ type FilterSetsProps = {
 const DEFAULT_FILTER_SET_NAME = t('New filter set');
 
 const FilterSets: React.FC<FilterSetsProps> = ({
+  currentDataMask,
```

Comment:
Would be nice to have these in the same style; below we have `dataMaskApplied`, maybe this could be `dataMaskSelected`?

Reply:
It's not selected it's applied, because it's different
it can be selected but not applied